docs/application-connector/08-11-rotate-root-ca.md

```diff
@@ -30,63 +30,63 @@ To successfully rotate a soon-to-expire CA certificate, replace it with a new ce
    kubectl -n kyma-integration get secret connector-service-app-ca -o=jsonpath='{.data.ca\.key}' | base64 --decode > ca.key
    ```
 
-1. Generate a new certificate for the key you obtained and save it to the `new-ca.crt` file.
+2. Generate a new certificate for the key you obtained and save it to the `new-ca.crt` file.
 
    ```bash
    openssl req -new -key ca.key -x509 -sha256 -days {TTL_DAYS} -nodes -out new-ca.crt
    ```
 
    >**NOTE:** Use the `-days` flag to set the TTL (Time to live) of the newly generated certificate.
 
-1. Encode the newly created certificate with base64.
+3. Encode the newly created certificate with base64.
   
    ```bash
    cat new-ca.crt | base64
    ```
 
-1. Replace the old certificate in the Connector Service CA Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate.
+4. Replace the old certificate in the Connector Service CA Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate.
```

Comment:
Just to make sure - this `ca.crt` shouldn't be replaced with `cacert`, is that right?

Reply:
It shouldn't. This is due to some changes introduced by another team and going with another name.